### PR TITLE
Add vita links to eligibility page

### PR DESCRIPTION
**Pull request by embarnard**

### CODE REVIEW

**Comment by embarnard** [comment]
Screenshot 2024-03-13 at 3 37 00 PM

File: app/views/state_file/questions/eligible/edit.html.erb [hunk 1]
```diff
@@ -34,6 +34,10 @@
       <div class="with-top-separator">
         <p class="text--bold"><%=t('.want_to_claim_heading') %></p>
         <p><%=t('.want_to_claim_vita', state_name: state_name) %></p>
+        <p>
+          <% vita_link = params[:us_state] == "az" ? "https://airtable.com/appnKuyQXMMCPSvVw/pag0hcyC6juDxamHo/form" : "https://airtable.com/appQS3abRZGjT8wII/pagtpLaX0wokBqnuA/form" %>
```
Comment (rickreyhsig) [inline app/views/state_file/questions/eligible/edit.html.erb:38]: It seems like we are duplicating these links in the codebase.  Might be good to make them a constant somewhere that can just be referenced everywhere.

File: app/controllers/state_file/questions/eligible_controller.rb [hunk 2]
```diff
@@ -5,6 +5,7 @@ class EligibleController < QuestionsController
 
       def edit
         super
+        @vita_link = vita_link
```
Comment (embarnard) [inline app/controllers/state_file/questions/eligible_controller.rb:8]: @rickreyhsig good point about using shared concerns! I forgot about the `OtherOptionsLinksConcern` that has those airtable links

Comment (rickreyhsig) [reply app/controllers/state_file/questions/eligible_controller.rb:8]: Perfect!

**Review by rickreyhsig — APPROVED** [review]

**Pull request merged**